class2.md

```diff
@@ -0,0 +1,287 @@
+## Concepts in Machine Learning Class 2: Supervised Learning
+
+### Objectives
+
+Welcome to class 2 of Concepts in Machine Learning!
+
+In the last class, we touched on all of the main concepts that we will be covering in this course including supervised and unsupervised learning, exploratory data analysis, and ethics.
+
+By the end of this class you should be able to:
+
+* Define and understand the limitations of supervised learning
+* Differentiate between regression and classification
+* Assess whether or not supervised learning is an appropriate tool for a question
+* Understand basic applications of supervised learning (linear regression, logistic regression)
+* Create supervised learning problem statements
+
+### Review! What is machine learning?
```

Comment:
Condense the review sections into a single subheading. For example, remove the actual data being referenced in the anatomy section.  I don't recommend exactly duplicating content between lessons, mostly because it's a huge pain if you have to update in more than one place. A paragraph at most should suffice.